open-apparel-registry/open-apparel-registry
# Create OGR staging environment

## Overview
This PR creates a staging environment for the ogr/develop branch. 

Connects #1863


## Notes
work for this issue started with manually creating a zone for the domain `oshstaging.openapparel.org` in route53, and delegating the record for oshstaging from the primary zone for `openapparel.org` to the name servers assigned to the newly created zone.

* r53_public_hosted_zone was set to `oshstaging.openapparel.org` in the .tfvars file
* r53_private_hosted_zone was set to `osh.internal` in the .tfvars file
* S3 bucket names have been updated to not conflict with OAR staging
* the public_hosted_zone needed to be updated in a few other files it was specified in

## Testing Instructions
* open a docker image capable of running terraform with `docker-compose -f docker-compose.ci.yml run --rm terraform`
* run `./scripts/infra plan`
* review the output to confirm that no existing resources will be changed
* 
## Checklist
